docs/src/main/sphinx/sql/explain.rst

```diff
@@ -84,6 +84,286 @@ Logical plan::
+                                           "name":"Aggregate",
+                                           "identifier":{
+                                              "type":"PARTIAL",
+                                              "groupingKeys":"[regionkey]",
+                                              "hashSymbol":""
+                                           },
+                                           "outputs":[
+                                              {
+                                                 "symbol":"regionkey",
+                                                 "type":"bigint"
+                                              },
+                                              {
+                                                 "symbol":"count_0",
+                                                 "type":"bigint"
+                                              }
+                                           ],
+                                           "details":[
+                                              "count_0 := count(*)"
+                                           ],
+                                           "estimates":[
+
+                                           ],
+                                           "children":[
+                                              {
+                                                 "id":"0",
+                                                 "name":"TableScan",
+                                                 "identifier":{
+                                                    "table":"hive:tpch_sf1_orc_part:nation"
+                                                 },
+                                                 "outputs":[
+                                                    {
+                                                       "symbol":"regionkey",
+                                                       "type":"bigint"
+                                                    }
+                                                 ],
+                                                 "details":[
+                                                    "regionkey := regionkey:bigint:REGULAR"
+                                                 ],
+                                                 "estimates":[
+                                                    {
+                                                       "outputRowCount":25,
+                                                       "outputSizeInBytes":225,
+                                                       "cpuCost":225,
+                                                       "memoryCost":0,
+                                                       "networkCost":0
+                                                    }
+                                                 ],
+                                                 "children":[],
+                                                 "remoteSources":[]
```

Comment:
Why is this here? "remoteSources" is only meaningful in the distributed plan, and only for exchange nodes.

Reply:
We can't remove `remoteSources` from `JsonRenderNode` because it is used in Trino's LivePlan UI. It was introduced in this commit: https://github.com/trinodb/trino/commit/ae5f23e3daade05cbbf2bd678bf74498a0fe375d#diff-c1094198b4b27865839dd6c6302e0bfa4d81d3310f88b245e5d85ba0d96f63b9

Reply:
That should be adjusted to be able to handle a missing `remoteSources` field.

Reply:
I've removed "remoteSources"